docs/src/forc_client.md

```diff
@@ -1,3 +1,45 @@
 # forc-client
 
 Forc plugin for interacting with a Fuel node.
+
+## Signing transactions using `forc-wallet` CLI
+
+To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without utxo validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
```

Comment:
This paragraph mentions UTXO validation being disabled.
My initial reaction is "what is that? am I using that? how do I check? should I be using that?" etc. There should be a hyperlink here to some relevant page which explains what this is so that a user knows what the default state is and whether they want this enabled or disabled.

Reply:
I couldn't find any written document describing UTXO validation so added a short note section.